### DISCUSSION TITLE
! HELP please ! KSampler mixed dtype (CPU): expect parameter to have scalar type of Float

### DISCUSSION THREAD

**Discussion by neipr3s** [comfyanonymous/ComfyUI#6092] [Q&A]
Hello why am I getting this error from the K Sampler, it alternates between this and 'mat1 and mat2 must have same dtype, but float and bfloat16'

I have been stuck on this for almost 24 hours, I can't get my head around it, please check my workflow. 
![Screenshot 2024-12-17 202726](https://github.com/user-attachments/assets/95c01e8a-853f-462d-baaf-831e74fad1cc)

**Comment by ltdrdata** [comment]
Duplicate: https://github.com/comfyanonymous/ComfyUI/issues/6091
